src/status_im/node/core.cljs

```diff
@@ -50,15 +50,11 @@
       les-topic (assoc les-topic {:Min 2, :Max 2}))))
 
 (defn- get-base-node-config [config]
-  (let [initial-props @(re-frame/subscribe [:initial-props])
-        status-node-port (get initial-props :STATUS_NODE_PORT)]
-    (cond-> (assoc config
-                   :Name "StatusIM")
-      config/dev-build?
-      (assoc :ListenAddr ":30304"
-             :DataDir (str (:DataDir config) "_dev"))
-      status-node-port
-      (assoc :ListenAddr (str ":" status-node-port)))))
```

Comment:
@cammellos do you know what is `:ListenAddr` ?

Reply:
It's the port the client binds, on desktop you probably wanted to run multiple instances, so you passed it as an ENV variable